integration/networktest/traffic/runner.go

```diff
@@ -0,0 +1,160 @@
+package traffic
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/obscuronet/go-obscuro/integration/networktest"
+)
+
+// Runner is used to simulate traffic against a durationTrafficTest network
```

Comment:
why only against a durationTrafficTest?

Reply:
This is the traffic test runner, the durationTrafficTest is how it's implemented I guess. Just simulates activity for a duration then runs the configured verifiers against it. But I think maybe this stuff needs a rethink since the interface doesn't seem to be intuitive.